@here/harp-examples/src/datasource_features_polygons.ts

```diff
@@ -281,7 +280,6 @@ export namespace PolygonsFeaturesExample {
         const mapView = new MapView({
             canvas,
             projection: sphereProjection,
-            clipPlanesEvaluator: new TopViewClipPlanesEvaluator(300000),
```

Comment:
Here we will need to pass this visualization geometry height into (300000) DataSource as (default) maxGeometryHeight.

Reply:
done